File: DSA with python/fundamental/fibonacci.py
def getFibonacci(num):
    if num==0:
        return [0];
    if num==1:
        return [0,1];
    a = 0;
    b = 1
    ans = 0;
    ansArr = [];
    ansArr.append(a)
    ansArr.append(b)
    while(ans<num):
        ans = a+b
        if ans>num:
            break
        ansArr.append(ans);
        a = b;
        b = ans;
    
    return ansArr
    
    
finalAns = getFibonacci(10);
print(finalAns)

File: DSA with JS/fundamental/fibonacci.js
function getFibonacciSequence(num){
    let arr = [];
    if(num==0){
        arr.push(0);
        return arr;
    }
    if(num==1){
        arr.push(1);
        return arr
    }
    let a = 0;
    let b = 1;
    let sum = a;
    while(num>sum){
        arr.push(sum)
        sum = a+b;
        b = a;
        a = sum;
    }
    return arr
}


let ans = getFibonacciSequence(10);
console.log(ans)